client/src/component/customer/Menu.js

```diff
@@ -49,67 +50,38 @@ const ItemContent = ({ children }) => {
 };
 
 export default function Menu() {
-  const [earlyBreakfast, setEarlyBreakfast] = useState([]);
-  const [breakfast, setBreakfast] = useState([]);
-  const [supper, setSupper] = useState([]);
-  const [beverages, setBeverages] = useState([]);
+  const [menuList, setMenuList] = useState(null);
 
-  const getEarlyBreakfast = async () => {
-    setEarlyBreakfast(await getAllItem(1));
+  const getMenuList = async () => {
+    setMenuList(await getAllItem(1));
   };
 
   useEffect(() => {
-    getEarlyBreakfast();
+    getMenuList();
   }, []);
 
-  const getBreakfast = async () => {
-    setBreakfast(await getAllItem(2));
-  };
-
-  useEffect(() => {
-    getBreakfast();
-  }, []);
-
-  const getSupper = async () => {
-    setSupper(await getAllItem(3));
-  };
-
-  useEffect(() => {
-    getSupper();
-  }, []);
+  console.log(menuList);
 
-  const getBeverages = async () => {
-    setBeverages(await getAllItem(4));
-  };
-
-  useEffect(() => {
-    getBeverages();
-  }, []);
-
-  console.log(earlyBreakfast);
-  let arr = [earlyBreakfast, breakfast, supper, beverages];
-  // console.log(arr);
-
-  return (
+  return menuList ? (
     <>
-      {arr.map((category) => {
-        if (!category || !category.items) return null;
+      {menuList.map((category) => {
         return (
-          <Box bg="gray.50" spacing="8" key={category.categoryId}>
+          <Box bg="gray.50" spacing="8" key={category.id}>
             <Container maxW={"7xl"} py={16} as={Stack} spacing={12}>
               <Stack spacing={0} align={"center"}>
-                <Heading>{category.categoryName}</Heading>
+                <Heading>{category.categoryname}</Heading>
                 <Text>{category.description}</Text>
               </Stack>
-              <Stack
-                direction={{ base: "column", md: "row" }}
+              <SimpleGrid
+                // direction={{ base: "column", md: "row" }}
+                columns={[1, null, 4]}
                 spacing={{ base: 10, md: 4, lg: 10 }}
               >
-                {category.items.map((item) => {
+                {category.item.map((item) => {
                   return (
                     <Box key={item.id}>
                       <ItemContent>
-                        <Heading size="md">{item.item_name}</Heading>
+                        <Heading size="md">{item.itemName}</Heading>
```

Comment:
I suggest using a consistent case for all your JSON. Here it looks like you're using `item.itemName` (camel-case) but above you have `category.categoryname` (lower-case). Either option is fine but it should be consistent.